modyn/selector/selector_entrypoint.py

```diff
@@ -15,13 +14,16 @@
 logger = logging.getLogger(__name__)
 
 # We need to do this at the top because other dependencies otherwise set fork.
+# Especially the import of the SelectorGRPCServer will transivitly cause errors 
 try:
     mp.set_start_method("spawn")
 except RuntimeError as error:
     if mp.get_start_method() != "spawn" and "PYTEST_CURRENT_TEST" not in os.environ:
-        logger.error("Start method is already set to {}", mp.get_start_method())
+        logger.error(f"Start method is already set to {mp.get_start_method()}")
         raise error
 
+from modyn.selector.internal.grpc.selector_server import SelectorGRPCServer  # fmt: skip  # noqa  # isort:skip
```

Comment:
Don't ask me why, but when validating whether we can instantiate a metric in the pipeline schema, this fails unless the set the spawn start method before importing the gRPC server. I have no idea why this is only a problem with the selector and whatnot, but it is.

Reply:
ok nevermind in the other classes this has been solved already apparently, not sure why it just came up now in the selector.